docs/source/markdown/podman-image-prune.1.md

```diff
@@ -11,6 +11,9 @@ podman-image-prune - Remove all unused images from the local store
 you can delete all unused images.  Unused images are dangling images as well as any image that
 does not have any containers based on it.
 
+It skips the cache images i.e. intermediate images build during image building.
```

Comment:
Suggest: "The image prune command does not prune any cache image, i.e. intermediate images built during image creation"

If you don't take that, be sure to change "bulid" -> "built"